increased user instructions (#38)
fair-software/fairtally: README.rst

   :target: https://github.com/fair-software/fairtally/actions?query=workflow%3A%22PyPI%22
   :alt: PyPI Publish

Installation
------------

To install fairtally, do:

.. code-block:: console

  pip3 install fairtally

Usage
-----

> Apply fairtally to your favorite repositories like this:

Reviewer: ```suggestion
To use fairtally to check the compliance of multiple repositories, one can use the command below.
```

.. code-block:: console

  fairtally --html tally.html https://github.com/fair-software/fairtally https://github.com/fair-software/howfairis

> Then open the analysis in a web browser, for example Firefox:

Reviewer: ```suggestion
This command will generate a html report called `tally.html` which will contain the results of the checks for each repository.

Then open the analysis in a web browser, for example Firefox:
```

.. code-block:: console

  firefox tally.html

> The analysis will look something like this:

Reviewer: ```suggestion
The report will look similar to the example below:
```

.. image:: docs/_static/fairtally_example.png
  :target: https://fair-software.github.io/fairtally/_static/fairtally_example.html

> You can sort the table by clicking on the table headers. The purple plus signs on the left provide access to log messages of fairtally.

Reviewer: ```suggestion
You can sort the table by clicking on the table headers. The purple plus signs provide access to log messages of each repository.
```

Documentation
*************

Command line interface help can be retrieved with

.. code-block:: console

  fairtally --help

> Which then shows something like:

Reviewer: ```suggestion
The output of the command will be something like:
```

See the License for the specific language governing permissions and
limitations under the License.

Credits
*******

This package was created with `Cookiecutter <https://github.com/audreyr/cookiecutter>`_ and the `NLeSC/python-template <https://github.com/NLeSC/python-template>`_.

> Instructions for developers
> ***************************
> 
> The project setup is documented in `a separate document <project_setup.rst>`_. Feel free to remove this document (and/or the link to this document) if you don't need it.
> 
> Run tests (including coverage) with:
> 
> .. code-block:: console
> 
>   python setup.py test

Reviewer: ```suggestion
Instructions for developers
***************************

The developer documentation can be found in `README.dev.rst <README.dev.rst>`_.
```
